pkg/kubelet/dockershim/docker_container.go

```diff
@@ -387,12 +387,15 @@ func (ds *dockerService) ContainerStatus(_ context.Context, req *runtimeapi.Cont
 			// Note: Can't set SeLinuxRelabel
 		})
 	}
-	// Interpret container states.
+	// Interpret container states and convert time to unix timestamps.
 	var state runtimeapi.ContainerState
 	var reason, message string
+	ct, st, ft := createdAt.UnixNano(), int64(0), int64(0)
```

Comment:
would the better fix be to change the defaults in `getContainerTimestamps`? This way if docker returns bad times, kubernetes will report the same

Reply:
hm, never mind. I got it. I wonder what docker returns as a Finish time that is converted to `-6795364578871345152`?

Reply:
docker returns as a Finished time that is converted to `0`(`0001-01-01T00:00:00Z`)